framework/src/Volo.Abp.TextTemplating/Volo/Abp/TextTemplating/TextTemplating/AbpTemplateRenderer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.TextTemplating.Scriban;
+
+namespace Volo.Abp.TextTemplating
+{
+    public class AbpTemplateRenderer : ITemplateRenderer, ITransientDependency
```

Comment:
I think we have a design issue here. Core template rendering system should be render engine independent. Developer should add render engine packages to add render engines to the application.

So, we are actually dropping the Volo.Abp.TextTemplating package. But, for backward compability, we keep it and add a reference to Volo.Abp.TextTemplating.Scriban to keep the current behavior. However, this module will be completely empty, and probably will be deleted in the future
. 
All the code should be moved to Volo.Abp.TextTemplating.Core package

Volo.Abp.TextTemplating.Abstractions -> rename to Volo.Abp.TextTemplating.Core
Volo.Abp.TextTemplating.Razor -> depends on Volo.Abp.TextTemplating.Core
Volo.Abp.TextTemplating.Scriban -> depends on Volo.Abp.TextTemplating.Core


So, if I want to use Razor, I add Volo.Abp.TextTemplating.Razor to my application. If I want to use both engines, I add both of the razor and scriban packages.
If I only add Volo.Abp.TextTemplating.Core package, I can't use any engine, so I get exception when I try to render.